packages/account-v2/src/constants/errorMessages.tsx

```diff
@@ -0,0 +1,7 @@
+export const ERROR_MESSAGE = {
+    CLAIMED_DOCUMENT:
```

Comment:
cant we use API_ERROR_CODES.CLAIMED_DOCUMENT

Reply:
I've modified the API_ERROR_CODES object